src/main/java/org/broadinstitute/consent/http/service/DatasetService.java

```diff
@@ -186,23 +185,23 @@ public Consent createConsentForDataset(DatasetDTO dataset) {
 
     private boolean isConsentDataUseManualReview(DataUse dataUse) {
         return Objects.nonNull(dataUse.getOther()) ||
-              (Objects.nonNull(dataUse.getPopulationRestrictions()) && !dataUse
-                    .getPopulationRestrictions().isEmpty()) ||
-              (Objects.nonNull(dataUse.getAddiction()) && dataUse.getAddiction()) ||
-              (Objects.nonNull(dataUse.getEthicsApprovalRequired()) && dataUse
-                    .getEthicsApprovalRequired()) ||
-              (Objects.nonNull(dataUse.getIllegalBehavior()) && dataUse.getIllegalBehavior()) ||
-              (Objects.nonNull(dataUse.getManualReview()) && dataUse.getManualReview()) ||
-              (Objects.nonNull(dataUse.getOtherRestrictions()) && dataUse.getOtherRestrictions()) ||
-              (Objects.nonNull(dataUse.getPopulationOriginsAncestry()) && dataUse
-                    .getPopulationOriginsAncestry()) ||
-              (Objects.nonNull(dataUse.getPsychologicalTraits()) && dataUse
-                    .getPsychologicalTraits()) ||
-              (Objects.nonNull(dataUse.getSexualDiseases()) && dataUse.getSexualDiseases()) ||
-              (Objects.nonNull(dataUse.getStigmatizeDiseases()) && dataUse.getStigmatizeDiseases())
-              ||
-              (Objects.nonNull(dataUse.getVulnerablePopulations()) && dataUse
-                    .getVulnerablePopulations());
+                (Objects.nonNull(dataUse.getPopulationRestrictions()) && !dataUse
+                        .getPopulationRestrictions().isEmpty()) ||
+                (Objects.nonNull(dataUse.getAddiction()) && dataUse.getAddiction()) ||
```

Comment:
feel like there's a repeating pattern that should be put into a method.  Or maybe use a lamba to shorten, e.g
```suggestion
                Optional.ofNullable(dataUse.getAddiction()).orElse(false) ||
```

Probably makes sense to have the logic pulled into a method that checks for truthiness.  Justing thinking of making this giant boolean statement easier to read

Reply:
I agree it's not pretty, but this is a super old code path using objects we don't use anymore, and it doesn't look tested very well, so I'm wary to change it, and I think it will probably be removed soon.